wavefront/server/packages/flo_cloud/flo_cloud/postgres/postgres.py

```
@@ -124,6 +162,137 @@ def _write_cursor(self, operation: str = 'Write'):
+                query = query + sql.SQL(' LIMIT {n}').format(
+                    n=sql.Literal(capture_limit)
+                )
+            # Last, after ORDER BY and LIMIT, so only the returned rows are
+            # locked rather than everything the predicate matched.
+            query = query + sql.SQL(' FOR UPDATE')
+
+            cursor.execute(query, params or None)
+            before_rows = [dict(row) for row in cursor.fetchall()]
+            cursor.execute('RELEASE SAVEPOINT audit_before')
+            return before_rows, key_columns
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)